atlante/server/README.md

```diff
@@ -71,3 +101,67 @@ Returns:
    "status" : "requested" | "started" | "processing" | "completed",
 }
 ```
+
+5. `GET /jobs` will return the latest 100 jobs
```

Comment:
I recall looking at the code and the limit value can be passed in. Is this correct?

Reply:
No it's hard coded right now.